src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download \"%s\"</string>
+  <string name="error_syncing_format">Could not sync \"%s\"</string>
+  <string name="error_validating_format">Could not validate \"%s\"</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">%s downloaded</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
-  <string name="ignore_all_updates">Ignore all updates</string>
-  <string name="ignore_this_update">Ignore this update</string>
+  <string name="ignore_all_updates">Ignore all new versions</string>
+  <string name="ignore_this_update">Ignore this new version</string>
   <string name="incompatible_features_description">Missing features:</string>
-  <string name="incompatible_older_description">This version is older than installed on your device.
-    You\'ll have to uninstall the application to proceed.</string>
+  <string name="incompatible_older_description">Older than the one installed on your device.
+    You\'ll have to uninstall the application to install it.</string>
```

Comment:
I think it's better to not throw away "This version". This text is displayed in alert dialog, so it's important to not shorten sentences.

Reply:
I still think that it's a bad idea to shorten the sentence :/ Look at this:

![device-2020-07-06-100715](https://user-images.githubusercontent.com/24494863/86565459-62de0980-bf57-11ea-8f17-d9989faf738c.png)

Reply:
That is not good. The user only needs to know "Uninstall the current application before installing an older version".
Problem 1, it isn't really incompatible, it is only impossible to have two of the same. Incompatible applications is a concept elsewhere (?), and at least in modern F-Droid.
Problem 2, "Older than installed" is an error. Older than ___ installed" is missing a marked pronoun. The one/the ones, etc.

Reply:
> The user only needs to know "Uninstall the current application before installing an older version"

You're not right. User needs to know why this version is incompatible. It's because it's older than installed one. And then solution is suggested.

> Problem 1, it isn't really incompatible

This is how it's designed right now, either the application compatible (so it can be installed), or not. It's actually "incompatible" in sense it's "incompatible to the current package manager state", package manager won't install it.

> Problem 2, "Older than installed" is an error

I don't mind changing to "Older than installed one". You suggested to replace the whole phrase to "Older than installed one" before, while I think "This version is older than installed one" should be used.

>  Problem 3, it is too long.

There are even longer messages :) It's OK to place long messages into alert dialogs.